src/test/system_tests/ceph_s3_tests/s3-tests-lists/s3_tests_pending_list.txt

```diff
@@ -130,4 +130,5 @@ s3tests/functional/test_headers.py::test_object_create_bad_date_none_aws2
 s3tests/functional/test_headers.py::test_bucket_create_bad_authorization_invalid_aws2
 s3tests/functional/test_headers.py::test_bucket_create_bad_date_none_aws2
 s3tests_boto3/functional/test_s3.py::test_get_object_ifnonematch_good
-s3tests_boto3/functional/test_s3.py::test_get_object_ifmodifiedsince_failed
\ No newline at end of file
+s3tests_boto3/functional/test_s3.py::test_get_object_ifmodifiedsince_failed
```

Comment:
please also add the relevant unit tests

Reply:
You mean to say remove the relevant tests  from the pending list ?

Reply:
no sorry, the comment isn't relevant to this file, please just add regular unit tests for this issue.